libcxx/lib/abi/CHANGELOG.TXT

```diff
@@ -36,6 +36,14 @@ Version 21.0
   -------------
   Symbol added: _ZNSt3__113__hash_memoryEPKvm
 
+* [libc++][hardening] Introduce a dylib function to log hardening errors.
```

Comment:
We're not actually adding any symbols to the dylib in this patch, so this diff can go away (and also the ABI lists updates). The commit message should also be updated with that in mind.